src/main/webapp/resources/css/main.css

```diff
@@ -0,0 +1,271 @@
+@import url('https://fonts.googleapis.com/css?family=Bitter|Lato');
+
+body {
+    background-color: #edf4ff;
+    font-family: "Lato";
+    font-weight: 300;
+    font-size: 16px;
+    color: #555;
+
+    -webkit-font-smoothing: antialiased;
+    -webkit-overflow-scrolling: touch;
+
+    margin-top: 70px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: "Bitter";
+    font-weight: 300;
+    color: #333;
+}
+
+p {
+    line-height: 28px;
+    margin-bottom: 25px;
+}
+
+.centered {
+    text-align: center;
+}
+
+a {
+    color: #006400;
+}
+
+a:hover, a:focus {
+    color: #7b7b7b;
+    text-decoration: none;
+    outline: 0;
+}
+
+hr {
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid #ccc;
+    margin: 1em 0;
+    padding: 0;
+}
+
+.navbar {
+    font-weight: 400;
+    font-size: 14px;
+}
+
+.navbar-default {
+    background-color: #108d19;
+    border-color: #89d68c;
+}
+
+.navbar-default .navbar-brand {
+    color: #ffffff;
+    font-weight: bolder;
+    font-size: 22px;
+    letter-spacing: 1px;
+}
+
+#headerwrap {
+    background: url(../img/lamp1.png) no-repeat center top;
+    margin-top: -10px;
+    padding-top: 20px;
+    text-align: center;
+    background-attachment: relative;
+    /*background-position: center center;*/
+    min-height: 500px;
+    width: 100%;
+    /*background-size: 100%;*/
+
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    -webkit-background-size: cover;
+    background-size: cover;
+}
+
+#headerwrap h1 {
```

Comment:
It is not advised to use ids and CSS selectors. All CSS styles should better be assigned using classes. Correct here, and all the other instances as well